docs/admin/authorization.md

```diff
@@ -53,7 +53,7 @@ A request has the following attributes that can be considered for authorization:
   - what resource is being accessed (for resource requests only)
   - what subresource is being accessed (for resource requests only)
   - the namespace of the object being accessed (for namespaced resource requests only)
-  - the API group being accessed (for resource requests only)
+  - the API group being accessed (for resource requests only); the [core API group](../api.md#api-groups) is identified by the empty string
```

Comment:
If this is the first usage, use "an empty string" as opposed to "the empty string". "The" implies that it was mentioned before and could confuse users.